src/app/Main/Body/QueryPage/Drawer/Filter/SelectorList-Selector.component.tsx

```diff
@@ -35,9 +36,12 @@ export function SelectorListSelectorComponent(
     const { t } = useTranslation(["QueryPage"]);
 
     const filterAttribute: FilterType = mainFilterAttributes[props.index];
-    const filterValues: string[] = filter[filterAttribute];
+    let filterValues: string[] = []
+    if (!_.isEmpty(filter.selectedFilter)) {
+        filterValues = filter.selectedFilter[filterAttribute];
+    }

```

Comment:
Reject: I think you want to check if filter.selectedFilter[filterAttributes] is not undefined.